draft-ietf-ppm-dap.md

```diff
@@ -1462,19 +1478,38 @@ finishes and stores its output share for further processing as described in
 will be `leader_outbound` in the next round of continuation). For the latter
 case, the helper sets `prep_state` to `new_state`.
 
-The Leader then sends a POST request to the aggregation job URI used during
-initialization (see {{leader-init}}). The body is an `AggregationJob` message
-(see {{aggregation-helper-init}}) containing the `PrepareStep`s. The media type
-is set to "application/dap-aggregation-job".
+The Leader then instructs the Helper to advance the aggregation job to the round
+the Leader has just reached by sending the new `inbound` prepare messages to the
+Helper in a POST request to the aggregation job URI used during initialization
+(see {{leader-init}}). The body of the request is an
+`AggregationJobContinueReq`:
+
+~~~
+struct {
+  u16 round;
+  PrepareStep prepare_Steps<1..2^32-1>;
+} AggregationJobContinueReq;
+~~~
+
+`round` MUST be set to the Leader's current round and the `prepare_steps` field
```

Comment:
Consider describing the expectation on the sequence of `round` values, i.e. that the first `AggregationJobContinueReq` will have a round of 1 and subsequent requests will increment the round by 1 (I think).

To make the later text around "If the Leader's `round` is behind or more than one round ahead of the Helper's current round" work out, I think we also need to describe the Helper is considered to be at round 0 after aggregation initialization.

Reply:
Good point. I think I have made this clear in the latest commit.